metricbeat/module/etcd/metrics/_meta/testdata/metrics.plain

```diff
@@ -207,9 +207,9 @@ etcd_disk_backend_snapshot_duration_seconds_sum 0
 etcd_disk_backend_snapshot_duration_seconds_count 0
 # HELP etcd_disk_wal_fsync_duration_seconds The latency distributions of fsync called by wal.
 # TYPE etcd_disk_wal_fsync_duration_seconds histogram
-etcd_disk_wal_fsync_duration_seconds_bucket{le="0.001"} 0
-etcd_disk_wal_fsync_duration_seconds_bucket{le="0.002"} 0
-etcd_disk_wal_fsync_duration_seconds_bucket{le="0.004"} 0
+etcd_disk_wal_fsync_duration_seconds_bucket{le="0.001"} 1
+etcd_disk_wal_fsync_duration_seconds_bucket{le="0.002"} 1
+etcd_disk_wal_fsync_duration_seconds_bucket{le="0.004"} 1
```

Comment:
Umm, what error were causing the zero values? I think we should keep some of them in the test case.

Reply:
I mentioned above, when using the `expfmt` package to read and parse the values ending with 0 they are missing from the buckets. This happens only on windows 32 bit not all windows versions. I have updated the fields to 1 so we can have one of these tests still running on windows 32. There is a second test at the module level which will test with the `0` values as well (which has been skipped on windows 32 bit) but will run on all os's so we have that covered both ways.

Reply:
Interesting. But sounds good if we have this covered in other tests :+1: